functional-tests/src/test/resources/GebConfig.groovy

```diff
@@ -154,8 +154,8 @@ reportOnTestFailureOnly = false //true
 //in seconds
 //If unspecified, the values of 5 for timeout and 0.1 for retryInterval.
 waiting {
-    timeout = 30
-    retryInterval = 1
+    timeout = 60
+    retryInterval = 2
```

Comment:
12X the default timeout? Is that a symptom of our tests being in a dire situation or of the defaults being unrealistically low?

Reply:
I was doing some testing thinking that now that DEV will be relatively slower it may make more sense to increase timeouts on functional tests given how rapidly they run and jump from one scenario to next

Reply:
Im not too too certain these are _needed_ but not sure if I should turn them back down.

@thegentlemanphysicist any strong feelings?